react/features/filmstrip/functions.web.js

```diff
@@ -174,6 +177,33 @@ export function calculateThumbnailSizeForVerticalView(clientWidth: number = 0) {
     };
 }
 
+/**
+ * Calculates the size for thumbnails when in vertical view layout
+ * and the filmstrip is resizable.
+ *
+ * @param {number} clientWidth - The height of the app window.
+ * @param {number} filmstripWidth - The width of the filmstrip.
+ * @returns {{local: {height, width}, remote: {height, width}}}
+ */
+export function calculateThumbnailSizeForResizableVerticalView(clientWidth: number = 0, filmstripWidth: number) {
+    const horizontalMargin = VERTICAL_FILMSTRIP_MIN_HORIZONTAL_MARGIN + SCROLL_SIZE
+        + TILE_HORIZONTAL_MARGIN + STAGE_VIEW_THUMBNAIL_HORIZONTAL_BORDER;
+    const availableWidth = Math.min(
+        Math.max(clientWidth - horizontalMargin, 0),
+        filmstripWidth || DEFAULT_FILMSTRIP_WIDTH);
+
+    return {
+        local: {
+            height: DEFAULT_FILMSTRIP_WIDTH,
```

Comment:
I'm wondering why do we use the `DEFAULT_FILMSTRIP_WIDTH` here and not the ratios for local and remote videos defined in interface_config like `calculateThumbnailSizeForVerticalView`?

Reply:
Because when we resize the filmstrip we want to increase just the width of the tiles, not the height.
![image](https://user-images.githubusercontent.com/11474153/152749930-cbfbabbb-e5cf-4afd-8005-034837439fae.png)